www/templates/main-goals.html

```diff
@@ -75,11 +82,11 @@
               <p class="stats-name">Health</p>
             </div>
           </div>
-          <div>
+          <div id="stats-wrapper">
             <i id="habitica-star-icon" class="icon ion-android-star"></i>
             <progress class="golden" value="{{profile.stats.exp}}" max="{{profile.stats.toNextLevel}}"></progress>
             <div class="stats-value-wrapper">
-              <p class="stats-value">{{profile.stats.exp}}/{{profile.stats.toNextLevel}}</p>
+              <p class="stats-value">{{exp}}/{{toNextLevel}}</p>
```

Comment:
why is this `{{exp}}` and `{{toNextLevel}}` and not `{{profile.stats.exp}}` and `{{profile.stats.toNextLevel}}`?